librz/bin/format/dmp/dmp_specs.h

```diff
@@ -135,8 +135,8 @@ typedef struct {
 	ut64 FirstPage;
 	ut64 TotalPresentPages;
 	ut64 Pages;
-	ut8 Bitmap[1];
```

Comment:
I recommend here keeping the name of the parameter as a bitmap, and all related variable names, since it's really is a bitmap.